qiskit/opflow/__init__.py

```diff
@@ -19,7 +19,7 @@
 
 Operators and State functions are the building blocks of Quantum Algorithms.
 
-A library for Quantum Algorithms & Applications is more than a collection of
+A library for Quantum Algorithms & Applications (QA&A) is more than a collection of
```

Comment:
This is unrelated. I just saw it around and noticed that _QA&A_ is not a very common acronym.

Reply:
It'd probably be better to revert this, especially with opflow deprecated.  It's a problem that's existed forever (and capitalising unexpected words in a sentence like this is a not-altogether-unheard-of way of implicitly defining an acronym anyway).

Reply:
done in 519028529dab09960efba9388b9a2dd77c86aedd